bots/github/test-cache

```diff
@@ -58,5 +59,30 @@ class TestCache(unittest.TestCase):
         result = c.read("path")
         self.assertEqual(result, other)
 
+    def testCurrent(self):
+        c = cache.Cache(self.directory, lag=3)
+
+        self.assertFalse(c.current("resource"))
+
+        c.write("resource", { "value": 1 })
+        c.write("resource2", { "value": 2 })
+        self.assertTrue(c.current("resource"))
+        self.assertTrue(c.current("resource2"))
+
+        time.sleep(2)
+        self.assertTrue(c.current("resource"))
+        self.assertTrue(c.current("resource2"))
+
+        c.mark()
+        self.assertFalse(c.current("resource"))
+
+        time.sleep(2)
+        self.assertFalse(c.current("resource2"))
```

Comment:
This should get two test  cases, one for `lag=3` and `sleep(4)` to test the automatic obsoletion, and another one with calling `mark()` for manual obsoletion. Like that you don't test the former properly.